app/lib/common/widgets/html_editor/components/mention_item.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:acter/common/utils/constants.dart';
+import 'package:acter/common/widgets/html_editor/models/mention_type.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:flutter/material.dart';
+
+class MentionItem extends StatelessWidget {
```

Comment:
Mention list UI item component which gets displayed as part of mention menu list.